spec/pluto/grey_image_spec.cr

```diff
@@ -0,0 +1,63 @@
+require "../spec_helper"
+
+def grey_image : Pluto::GreyImage
+  grey = [
+    255, 0u8, 255,
+    0u8, 1u8, 0u8,
+    255, 255, 255,
+  ] of UInt8
+  width = 3
+  height = 3
+
+  Pluto::GreyImage.new(grey, width, height)
+end
+
+def sample_image
+  Pluto::GreyImage.open("lib/pluto_samples/pluto.ppm")
+end
+
+def hash(content : String)
+  Digest::SHA1.hexdigest(content)
+end
+
+describe Pluto::GreyImage do
```

Comment:
I didn't want to add the dependency without asking here first: what are thoughts around [spectator](https://gitlab.com/arctic-fox/spectator)? I've enjoyed using it for personal projects and find it provides a lot of testing nice-to-haves (such as the `expect` macro which shows what's being run as well as what the differences were, rather than the current `should` that only shows the differences).

Reply:
Well, there would have to be a reason to introduce Spectator. I'm not sure if showing what's being run is a good one (unless I'm misunderstanding).

Reply:
I don't think spectator introduces anything as a must have, but it does also have a nice [suite of features](https://gitlab.com/arctic-fox/spectator/-/wikis/home) that I find makes testing more streamlined. 

The specific bit I was thinking about is mentioned [here](https://gitlab.com/arctic-fox/spectator/-/tree/master#assertions), but I think regular `should`s now show the expression that failed, not just the result.

Regardless, sounds like spectator isn't a good fit at this time :grin:

Reply:
The features look good, but yeah, I think it's better to prioritize simplicity for now. :)